### ISSUE TITLE
Conclusion: don't install krisawzm/critical-css v2.1.1

### ISSUE THREAD

**Issue by jesus997** [kalfheim/critical-css#26]
When trying to install this package I got the following error, am I doing something wrong?

`[0] % composer require krisawzm/critical-css
Using version ^2.1 for krisawzm/critical-css
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Conclusion: don't install krisawzm/critical-css v2.1.1
    - Conclusion: remove laravel/framework v5.7.14
    - Installation request for krisawzm/critical-css ^2.1 -> satisfiable by krisawzm/critical-css[v2.1.0, v2.1.1].
    - Conclusion: don't install laravel/framework v5.7.14
    - krisawzm/critical-css v2.1.0 requires illuminate/view 5.0.*|5.1.*|5.2.*|5.3.*|5.4.* -> satisfiable by illuminate/view[5.0.x-dev, 5.1.x-dev, 5.2.x-dev, 5.3.x-dev, 5.4.x-dev, v5.0.0, v5.0.22, v5.0.25, v5.0.26, v5.0.28, v5.0.33, v5.0.4, v5.1.1, v5.1.13, v5.1.16, v5.1.2, v5.1.20, v5.1.22, v5.1.25, v5.1.28, v5.1.30, v5.1.31, v5.1.41, v5.1.6, v5.1.8, v5.2.0, v5.2.19, v5.2.21, v5.2.24, v5.2.25, v5.2.26, v5.2.27, v5.2.28, v5.2.31, v5.2.32, v5.2.37, v5.2.43, v5.2.45, v5.2.6, v5.2.7, v5.3.0, v5.3.16, v5.3.23, v5.3.4, v5.4.0, v5.4.13, v5.4.17, v5.4.19, v5.4.27, v5.4.36, v5.4.9].
    - don't install illuminate/view 5.0.x-dev|don't install laravel/framework v5.7.14
    - don't install illuminate/view 5.1.x-dev|don't install laravel/framework v5.7.14
    - don't install illuminate/view 5.2.x-dev|don't install laravel/framework v5.7.14
    - don't install illuminate/view 5.3.x-dev|don't install laravel/framework v5.7.14
    - don't install illuminate/view 5.4.x-dev|don't install laravel/framework v5.7.14
    - don't install illuminate/view v5.0.0|don't install laravel/framework v5.7.14
    - don't install illuminate/view v5.0.22|don't install laravel/framework v5.7.14
    - don't install illuminate/view v5.0.25|don't install laravel/framework v5.7.14
    - don't install illuminate/view v5.0.26|don't install laravel/framework v5.7.14
    - don't install illuminate/view v5.0.28|don't install laravel/framework v5.7.14
    - don't install illuminate/view v5.0.33|don't install laravel/framework v5.7.14
    - don't install illuminate/view v5.0.4|don't install laravel/framework v5.7.14
    - don't install illuminate/view v5.1.1|don't install laravel/framework v5.7.14
    - don't install illuminate/view v5.1.13|don't install laravel/framework v5.7.14
    - don't install illuminate/view v5.1.16|don't install laravel/framework v5.7.14
    - don't install illuminate/view v5.1.2|don't install laravel/framework v5.7.14
    - don't install illuminate/view v5.1.20|don't install laravel/framework v5.7.14
    - don't install illuminate/view v5.1.22|don't install laravel/framework v5.7.14
    - don't install illuminate/view v5.1.25|don't install laravel/framework v5.7.14
    - don't install illuminate/view v5.1.28|don't install laravel/framework v5.7.14
    - don't install illuminate/view v5.1.30|don't install laravel/framework v5.7.14
    - don't install illuminate/view v5.1.31|don't install laravel/framework v5.7.14
    - don't install illuminate/view v5.1.41|don't install laravel/framework v5.7.14
    - don't install illuminate/view v5.1.6|don't install laravel/framework v5.7.14
    - don't install illuminate/view v5.1.8|don't install laravel/framework v5.7.14
    - don't install illuminate/view v5.2.0|don't install laravel/framework v5.7.14
    - don't install illuminate/view v5.2.19|don't install laravel/framework v5.7.14
    - don't install illuminate/view v5.2.21|don't install laravel/framework v5.7.14
    - don't install illuminate/view v5.2.24|don't install laravel/framework v5.7.14
    - don't install illuminate/view v5.2.25|don't install laravel/framework v5.7.14
    - don't install illuminate/view v5.2.26|don't install laravel/framework v5.7.14
    - don't install illuminate/view v5.2.27|don't install laravel/framework v5.7.14
    - don't install illuminate/view v5.2.28|don't install laravel/framework v5.7.14
    - don't install illuminate/view v5.2.31|don't install laravel/framework v5.7.14
    - don't install illuminate/view v5.2.32|don't install laravel/framework v5.7.14
    - don't install illuminate/view v5.2.37|don't install laravel/framework v5.7.14
    - don't install illuminate/view v5.2.43|don't install laravel/framework v5.7.14
    - don't install illuminate/view v5.2.45|don't install laravel/framework v5.7.14
    - don't install illuminate/view v5.2.6|don't install laravel/framework v5.7.14
    - don't install illuminate/view v5.2.7|don't install laravel/framework v5.7.14
    - don't install illuminate/view v5.3.0|don't install laravel/framework v5.7.14
    - don't install illuminate/view v5.3.16|don't install laravel/framework v5.7.14
    - don't install illuminate/view v5.3.23|don't install laravel/framework v5.7.14
    - don't install illuminate/view v5.3.4|don't install laravel/framework v5.7.14
    - don't install illuminate/view v5.4.0|don't install laravel/framework v5.7.14
    - don't install illuminate/view v5.4.13|don't install laravel/framework v5.7.14
    - don't install illuminate/view v5.4.17|don't install laravel/framework v5.7.14
    - don't install illuminate/view v5.4.19|don't install laravel/framework v5.7.14
    - don't install illuminate/view v5.4.27|don't install laravel/framework v5.7.14
    - don't install illuminate/view v5.4.36|don't install laravel/framework v5.7.14
    - don't install illuminate/view v5.4.9|don't install laravel/framework v5.7.14
    - Installation request for laravel/framework (locked at v5.7.14, required as 5.7.*) -> satisfiable by laravel/framework[v5.7.14].

Installation failed, reverting ./composer.json to its original content.
`

I'm using **laravel 5.7**

**Comment by ghost** [comment]
Hi @jesus997 
Did you get this to work? - I get the same error...

**Comment by jesus997** [comment]
> Hi @jesus997
> Did you get this to work? - I get the same error...

Nope, I tried to update the dependencies myself (#27) but it did not work for me.
